tests/test_schema_types.py

```diff
@@ -207,3 +256,90 @@ def __init__(
     )
 
     assert expected == actual
+
+
+def test_mapping_overrides():
+    """
+    Test mapping overrides
+    """
+
+    from pg2avro.pg2avro import Column
+
+    table_name = "test_table"
+    namespace = "test_namespace"
+
+    columns = [
+        Column(name="int_to_string", type="int"),
+        Column(name="string_to_numeric", type="string"),
+        Column(name="not_overriden", type="int"),
+        Column(name="numeric_to_float", type="numeric"),
+    ]
+    overrides = {
```

Comment:
Can you please test if it works correctly with arrays (if some valid constellation for that case even exists) ?

Reply:
Added array tests both ways - its kinky but possible so better have it covered :)